installation/curl-autoconf-archive.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+# Description: This script downloads and installs the autoconf-archive macros
+#   necessary to configure this project. This is invoked for distro builds where
+#   the necessary package is not available to install, and is also useful to
+#   hand run under certain circumstances.
+#   Should be run from the top level directory
+#
+
+# autoconf-archive url
+autoconf_archive_url="http://git.savannah.gnu.org/gitweb/?p=autoconf-archive.git;a=blob_plain;f=m4"
```

Comment:
@grahamwhaley @jodh-intel , not problem here, I was curious to  be a  consistent with the versions.txt, it already includes a version for "autoconf_archive_version", and in case of that happen,  we potentially could know about it and  fix it quickly